src/main/resources/assets/botania/lang/zh_cn.lang

```diff
@@ -3886,4 +3886,4 @@ botania.subtitle.terraPickMode=泰拉粉碎者升级
 botania.subtitle.terrasteelCraft=泰拉钢制作完成
 botania.subtitle.thermalily=炽玫瑰消耗熔岩
 botania.subtitle.unholyCloak=罪恶斗篷 激活
-# No translation # botania.subtitle.way=
+# No translation, and please, any attempt of translating this entry will be considered as vandalism to zh_cn.lang # botania.subtitle.way=
```

Comment:
Adding `#` at the line-beginning makes it a comment, and thus fallback to `en_us.lang`.